### ISSUE TITLE
LVIS few shot finetuning configs

### ISSUE THREAD

**Issue by hanoonaR** [ashkamath/mdetr#75]
Hi, 
Thank you for sharing your great work on open source. I have some questions regarding the fine-tuning on LVIS few shot long-tailed OD. In the readme, it has been provided that the fine-tuning was done on 64 GPU's, but according to the fine-tuning commands, the batch size or learning rate has not been modified. So does that mean the effective batch size for fine-tuning on LVIS is 128 (64 GPUs x 2 default BS) ? 

Also it has been specified that the number of epoch depends on the size of the subset, 1, 10 and 100. I assume, the provided number of epochs 150 is for subset 10. Does any other config like BS or LR vary with this? It would be great if you could provide the number of epochs for 1 and 100 as well. 

Thank you in advance.
